html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <label for="mail">Email</label><br>
+    <input type="email" id="mail" name="user_email" /><br>
+    <label for="options">Why is coding cool?</label><br>
```

Comment:
This should just be `option` instead of `options`. You don't have any element with an `id` of `options`

Reply:
Got it. Thank you!

Reply:
<img width="1512" alt="Screen Shot 2023-05-15 at 3 57 40 PM" src="https://github.com/loganlee-lth/c0523-code-solutions/assets/36302145/2b9ce7a6-56cc-44c8-9e00-16ed7074980e">